src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.iOS.cs

```diff
@@ -1506,13 +1511,25 @@ private void UpdateLayoutForReordering()
 			if (_reorderingState is { } reorderingState && reorderingState.LayoutAttributes is { } draggedAttributes)
 			{
 				var dropTargetAttributes = GetLayoutAttributesUnderPoint(reorderingState.Location);
+				if (dropTargetAttributes is null)
+				{
+					// If we have a reorderingState, it means that the pointer is still over the LV.
```

Comment:
Could it also be before the first item?

Reply:
Do we support to vertically center items in a ListView?

Reply:
not to my knowledge, but what if, eg, there's a Padding set on the ListView or its ItemsPresenter?

Reply:
Good point ... fixed!